pom.xml

```
@@ -58,9 +59,9 @@
 
         <!-- Tests -->
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>${junit.version}</version>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.9.3</version>
```

Would you mind to update the property instead?
```suggestion
            <version>${junit.version}</version>
```